docs/modules/ROOT/pages/config-safe.adoc

```diff
@@ -1,37 +1,29 @@
 = Securing Cluster Configuration Using the Config Safe
 
-:description: This guide shows you how to store cluster configuration in the encrypted storage.
+:description: This guide shows you how to store cluster configuration in encrypted storage.
 
 {description}
 
 == Introduction
 
-Cluster configuration is one of the central ideas of Hazelcast CLC.
-CLC provides a convenient way of connecting to different clusters by their names.
-Although CLC provides several ways of creating new cluster configurations, it is also very easy to add and manipulate them.
-They are just YAML files in `$CLC_HOME/configs` directory after all!
+By default, cluster configuration is stored as plain text YAML files in the `$CLC_HOME/configs` directory. This provides convenient access, but sensitive data such as user credentials and SSL key passwords should be stored more securely. CLC's *Config Safe* provides encrypted storage for cluster configuration that includes sensitive data.
```

Comment:
Would it be possible to make it read so that the Config Safe is more of an option rather than a rule to store credentials?